src/registry/domain/plugins-initialiser.js

```diff
@@ -81,27 +81,28 @@ module.exports.init = function(pluginsToRegister, callback) {
   };
 
   const loadPlugin = function(plugin, cb) {
+    const done = _.once(cb);
```

Comment:
This allows robustness against plugins that would erroneously call the done() callback multiple times:
https://github.com/opencomponents/oc-hobknob/blob/master/index.js#L25